airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -436,6 +436,62 @@
                             "required": ["name", "format"],
                         },
                     },
+                    "use_file_transfer": {
+                        "airbyte_hidden": True,
+                        "default": False,
+                        "description": "Enable file-based bulk load",
+                        "title": "File Sync (Experimental)",
+                        "type": 'boolean'
+                    },
```

Comment:
Ditto comments elsewhere. Should we drop the boolean in favor of "delivery_method" approach? Having both makes it likely they could conflict.

Reply:
Yes, I will remove 🪓

Reply:
Removed.